rkt/trust.go

```diff
@@ -59,7 +59,15 @@ func init() {
 func runTrust(cmd *cobra.Command, args []string) (exit int) {
 	if flagPrefix == "" && !flagRoot {
 		if len(args) != 0 {
-			stderr.Print("--root required for non-prefixed (root) keys")
+			trustName := args[0]
+			cmdName := "trust"
+			
+			stdout.Printf(`Prevented potentially dangerous unbounded trust of all containers signed by keys from %s
+To continue trusting this root domain:
+	rkt %s --root %s PUBKEY (local key file, no discovery)
+Or, establish a smaller scope with a prefix:
+	rkt %s --prefix "example.com/hello"
```

Comment:
I also suspect this is not correct. From a quick look at the code, PUBKEY can be always specified. If omitted, it will be auto-discovered only if a prefix is specified.